packages/compiler/package.json

```diff
@@ -14,8 +21,6 @@
     "lint": "eslint src",
     "tc": "tsc --noEmit"
   },
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
```

Comment:
It's really necessary to change this?

Reply:
Now that there’s no ⁠dist, everything comes directly from the original ⁠src file. Otherwise, TypeScript tries to load the package from that file, but the linter workflow hasn’t built it yet and it fails.